src/documents/file_handling.py

```diff
@@ -144,26 +162,26 @@ def generate_filename(doc, counter=0, append_gpg=True, archive_filename=False):
                     replacement_text="-",
                 )
             else:
-                correspondent = "none"
+                correspondent = "-none-"
 
             if doc.document_type:
                 document_type = pathvalidate.sanitize_filename(
                     doc.document_type.name,
                     replacement_text="-",
                 )
             else:
-                document_type = "none"
+                document_type = "-none-"
 
             if doc.archive_serial_number:
                 asn = str(doc.archive_serial_number)
             else:
-                asn = "none"
+                asn = "-none-"
 
             # Convert UTC database date to localized date
             local_added = timezone.localdate(doc.added)
             local_created = timezone.localdate(doc.created)
 
-            path = settings.PAPERLESS_FILENAME_FORMAT.format(
+            path = settings.FILENAME_FORMAT.format(
```

Comment:
I think this got borked a bit in the merge.  Should be `filename_format` I think

Reply:
Don't get it

Reply:
As this, this will use the global filename format, instead of maybe the document specific format, as grabbed on line 140

Reply:
Of course! Will change.